templates/views/events.html

```diff
@@ -61,36 +78,31 @@ <h4 class="event_topic1">Past Events</h4>
 
             </div>
         </div>
+        {% if data.pastEvents|length %}
+        {% for event in data.pastEvents %}
+        {% if data.pastEvents.indexOf(event) %  2 !== 0 %}
         <div class="row row_padding">
             <div class="col-md-7 col-lg-7 col-sm-7 topic_padding">
-                <h4 class="event_topic4">React Ladies Bootcamp</h4>
-                <p class="event_topic2">Join over 1400 Women in Tech from across several African coutries on our Slack
-                    community to network, stay updated, get help, and grow in your career.</p>
+                <h4 class="event_topic3">{{event.title}}</h4>
+                <p class="event_topic2">{{event.description.md}}</p>
                 <button class="btn event_button">SEE EVENT GALLERY</button>
```

Comment:
All "see event gallery links" are missing the event title slug as well; i.e. the `href="/events/{{event.slug}}"`
Update in issue #24 changed the buttons to anchor tags but the `href` still needs to be updated.